quota/etcd/README.md

```diff
@@ -141,6 +145,20 @@ more tokens are restored per leaf batch. A value slightly higher than 1 (e.g.
 1.1) is recommended, so there is some protection against token leakage without
 too much compromise of the quota system in exceptional situations.
 
+`--quota_max_cache_entries` and `--quota_min_batch_size` are related to token
+caching. Some level of token caching (i.e., both flags having values > 0) is
+recommended to lessen the latency impact of rate limiting.
+
+`--quota_min_batch_size` is the least number of tokens acquired from etcd. If a
+particular request demands less tokens than the minimal batch size, the
+remaining tokens are kept in memory, saving new requests to etcd until those are
+insufficient.
```

Comment:
until those are consumed?

Reply:
Ta. I struggled to find a better word here.